src/contentScript/codeMirror5.ts

```diff
@@ -0,0 +1,49 @@
+import getSettings from "./getSettings";
+import type { IndentSettings } from "../settings";
+import type { PluginContext } from "./types";
+
+export default function(context: PluginContext) {
```

Comment:
`codeMirror5.ts` is a version of `indent.js`, migrated to TypeScript and disabled in the CodeMirror 6 (beta markdown) editor.